manifests/cf-manifest/deployments/040-cf-properties.yml

```diff
@@ -279,16 +279,20 @@ properties:
         client_key: (( grab secrets.consul_agent_key ))
         require_ssl: false
       dropsonde_port: (( grab properties.metron_agent.dropsonde_incoming_port ))
-      require_ssl: false
+      require_ssl: true
 
     converger:
       bbs:
         api_location: bbs.service.cf.internal:8889
         ca_cert: (( grab secrets.consul_ca_cert ))
-        client_cert: (( grab secrets.consul_agent_cert ))
-        client_key: (( grab secrets.consul_agent_key ))
+        client_cert: (( grab secrets.bbs_client_cert  ))
+        client_key: (( grab secrets.bbs_client_key  ))
```

Comment:
Although I think is fine for the current story use one certificate for all clients, in the future we should create a certificate for each client.